lib/reporters/verbose.js

```diff
@@ -33,6 +33,10 @@ VerboseReporter.prototype.test = function (test, runStatus) {
 		return '  ' + colors.skip('- ' + test.title);
 	}
 
+	if (test.failing) {
+		return ' ' + colors.error(test.title);
```

Comment:
This needs to be indented with another space. For consistency with the other messages we should also add a tick or something at the start, perhaps use `colors.error(figures.tick + ' ' + test.title)`?